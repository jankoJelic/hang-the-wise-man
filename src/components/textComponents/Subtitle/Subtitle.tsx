type Props = {
  text: string;
};

const Subtitle = ({ text }: Props) => (
  <h3 className="font-semibold text-mainText">{text}</h3>
);

export default Subtitle;
