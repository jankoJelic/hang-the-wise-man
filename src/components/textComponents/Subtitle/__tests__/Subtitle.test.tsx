import { render } from "@testing-library/react";
import renderer from "react-test-renderer";
import Subtitle from "../Subtitle";

it("renders correctly", () => {
  const tree = renderer.create(<Subtitle text="Test" />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders correct text", () => {
  const wrapper = render(<Subtitle text="Test" />);

  expect(wrapper.getByText("Test")).toBeInTheDocument();
});
