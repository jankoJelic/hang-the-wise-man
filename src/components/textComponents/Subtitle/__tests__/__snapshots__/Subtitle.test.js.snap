// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h3
  className="font-semibold text-mainText"
>
  Test
</h3>
`;
